### DISCUSSION TITLE
Default bufferline theme is confusing

### DISCUSSION THREAD

**Discussion by dpc** [helix-editor/helix#10270] [General]
I'm using `solarized_dark` (with minor tweaks that should irrelevant) and the bufferline with default (theme) settings looks like this:

![image](https://github.com/helix-editor/helix/assets/9209/0947e4a8-0559-4714-85b4-da61b9af8b09)

This is confusing. The "active" tab looks the same as background, making it not very distinguishable (especially when the active tab is the right-most one).

I've tweaked it locally with:

```yaml
"ui.bufferline" = { fg = "base03", bg = "base0" }
"ui.bufferline.active" = { fg = "base0", bg = "base03" }
"ui.bufferline.inactive" = { fg = "base03", bg = "base0" }
```

and now it looks:

![image](https://github.com/helix-editor/helix/assets/9209/bb2b8340-fbf6-4504-a46b-134de46a641a)

Now the active tab is the only distinct element, and looks a bit like an extension of the buffer view itself, which I consider very natural, and looks very tab-like when switching buffers:

![image](https://github.com/helix-editor/helix/assets/9209/7d0012c9-f43f-46a7-8493-0a02651e1503)

![image](https://github.com/helix-editor/helix/assets/9209/ddb4f00f-6b2c-4f99-a0b3-918b4aa4efed)

I did play a bit with making the active background slightly different than the buffer itself, but I actually didn't like it. And it would make a generic defaults for all themes harder.

Anyway - it seems that by tweaking the fallbacks in: https://github.com/helix-editor/helix/blob/d5fe08ddb8bf1408a1a92d39a63b51cd16d83255/helix-term/src/ui/editor.rs#L512 , it would be possible to achieve the same effect for all themes by default: 

* For inactive tabs and background use styling of the statusbar, so they blend and look like the statusline.
* For active tab use styling of the buffer view, so it looks like an extension/tab of the buffer.

**Comment by jhscheer** [comment]
> I've tweaked it locally with:
> 
> ```yaml
> "ui.bufferline" = { fg = "base03", bg = "base0" }
> "ui.bufferline.active" = { fg = "base0", bg = "base03" }
> "ui.bufferline.inactive" = { fg = "base03", bg = "base0" }
> ```

There's no `ui.bufferline.inactive`:
https://github.com/helix-editor/helix/blob/6f135e58a3e2897dc6566c26e3cea952f09380aa/helix-term/src/ui/editor.rs#L512-L530

**Comment by dpc** [comment]
@jhscheer Yeap. That line in my config is unnecessary. I'm not very familiar with the details there. I don't even know why `bg` works, where `.background` is used in that code. :shrug: . But it works for me so doesn't matter.

The main for the issue is - having good defaults out of the box.
